package.json

```diff
@@ -0,0 +1,77 @@
+{
+  "name": "nimble-infra",
+  "version": "0.0.0",
+  "description": "Nimble Infrastructure Template generator",
+  "author": "Nimblehq",
+  "bin": {
+    "nimble-infra": "./bin/run"
+  },
+  "homepage": "https://github.com/nimblehq/infrastructure-templates",
+  "license": "MIT",
+  "main": "dist/index.js",
+  "repository": "nimblehq/infrastructure-templates",
+  "files": [
+    "/bin",
+    "/dist",
+    "/npm-shrinkwrap.json",
+    "/oclif.manifest.json"
+  ],
+  "dependencies": {
+    "@oclif/core": "^1",
+    "@oclif/plugin-help": "^5",
+    "@oclif/plugin-plugins": "^2.0.1",
+    "generate-template-files": "^3.2.0",
+    "inquirer": "^8.2.4",
+    "inquirer-test": "^2.0.1"
+  },
+  "devDependencies": {
+    "@oclif/test": "^2",
+    "@types/chai": "^4",
+    "@types/inquirer": "^8.2.1",
+    "@types/mocha": "^9.0.0",
+    "@types/node": "^16.9.4",
+    "chai": "^4",
+    "eslint": "^7.32.0",
+    "eslint-config-oclif": "^4",
+    "eslint-config-oclif-typescript": "^1.0.2",
+    "globby": "^11",
+    "mocha": "^9",
+    "oclif": "^3",
+    "shx": "^0.3.3",
+    "ts-node": "^10.2.1",
+    "tslib": "^2.3.1",
+    "typescript": "^4.4.3"
+  },
+  "oclif": {
+    "bin": "nimble-infra",
+    "dirname": "nimble-infra",
+    "commands": "./dist/commands",
+    "plugins": [
+      "@oclif/plugin-help",
+      "@oclif/plugin-plugins"
+    ],
+    "topicSeparator": " ",
+    "topics": {
+      "hello": {
+        "description": "Say hello to the world and others"
```

Comment:
what is this @byhbt

Reply:
yup, removed, it's boilerplate code. https://github.com/nimblehq/infrastructure-templates/pull/22/commits/54e8f48ad3074556f80b8a4269d190266638a4aa